app/file.rb

```diff
@@ -80,7 +71,7 @@ def frontmatter?
 		end
 
 		def frontmatter
-			YAML.safe_load(open(path), [Date, Time])
+			YAML.safe_load(open(path), [Date, Time]) if frontmatter?
```

Comment:
I think we were also using this before for files that are 100% yaml -- the if might mess with that